tests/unittests/Foundation/ReferenceFoundation/TestNSURLResponse.mm

```diff
@@ -34,6 +34,7 @@
     NSURLResponse* res =
         [[[NSURLResponse alloc] initWithURL:url MIMEType:@"text/plain" expectedContentLength:0 textEncodingName:nil] autorelease];
     ASSERT_OBJCEQ_MSG(url, res.URL, @"should be the expected url");
+    ASSERT_OBJCEQ(@"path.txt", res.suggestedFilename);
 }
```

Comment:
These are the reference tests. We must make certain that they work on the reference platform, and we must make sure we document every change we make to them.

Where possible, do tests for this sort of stuff outside of the `Reference*` directories.